pkg/api/converterfromagentpoolonlyapi.go

```diff
@@ -130,12 +133,27 @@ func convertResourcePurchasePlanToV20180331AgentPoolOnly(api *ResourcePurchasePl
 	v20180331.Publisher = api.Publisher
 }
 
+func convertKubernetesConfigToEnableRBACV20180331AgentPoolOnly(kc *KubernetesConfig) *bool {
+	if kc == nil {
+		return common.BoolPtr(false)
+	}
+	// Only if both KubernetesConfig.EnableRbac and KubernetesConfig.EnableSecureKubelet true
+	// We think EnableRBAC is on
+	if kc != nil && kc.EnableRbac != nil && *kc.EnableRbac &&
+		kc.EnableSecureKubelet != nil && *kc.EnableSecureKubelet {
```

Comment:
I still think that `EnableSecureKubelet ` should not factor into converting back to versioned API model.  `if  *kc.EnableRbac`  then it should be enough to say that RBAC is enabled. You can also always log a warning line that `EnableSecureKubelet` was not enabled.

Reply:
I don't have strong opinion on this. Updated only use `*kc.EnableRbac` to deduct the versioned api model.